invenio_oauthclient/handlers.py

```diff
@@ -234,20 +235,18 @@ def authorized_signup_handler(resp, remote, *args, **kwargs):
 
         if user is None:
             # Auto sign-up if user not found
-            user = oauth_register(account_info)
-            if user is None:
```

Comment:
Hmm, that's questionable. I would like to see the registration form with pre-populated values, but I can image that in some use-cases it might be confusing for the end-user.